### PR TITLE
Add doc for creating an ISS provision

### PR DESCRIPTION
**Pull request by markhallen**
### Context

Documentation regarding the creation of an ISS Provision

### Changes proposed in this pull request

Documentation only

### Guidance to review

Follow the steps to confirm accuracy

### CODE REVIEW

**Comment by markhallen** [comment]
[Create accounts for Social Care and ISS](https://trello.com/c/uiS9XCvL/1919-create-accounts-for-social-care-and-iss)

File: docs/support/add_iss_provision.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+```ruby
+lafp = la.create_iss_provision!
+```
+
+Here is the method:
+[def create_iss_provision!](https://github.com/DFE-Digital/get-help-with-tech/blob/05a30daf5e09475b2d6cccedd5178e11a028647b/app/models/local_authority.rb#L23-L30) This will return the ISS if it already exists.
+
+## Update the details for the ISS Provision
+
+Set the contact details on `lafp`:
```
Comment (nigel-lowry) [inline docs/support/add_iss_provision.md:39]: Is this the delivery address, contact address or what? I'm still hazy on this

File: docs/support/add_iss_provision.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+You can also check the specs [here](https://github.com/DFE-Digital/get-help-with-tech/blob/05a30daf5e09475b2d6cccedd5178e11a028647b/spec/models/local_authority_spec.rb#L6)
+
+## Post creation
+
+Ensure that you have the `la` and `lafp` set correctly:
+
+```ruby
+urn=845;la=LocalAuthority.find_by_gias_id("#{urn}");lafp=School.find_by_provision_urn("ISS#{urn}")
```
Comment (nigel-lowry) [inline docs/support/add_iss_provision.md:68]: What should you look for here?

File: docs/support/add_iss_provision.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+### Updating allocations
+
+```ruby
+lafp.std_device_allocation.update!(allocation: 58, cap:58) # laptops
+lafp.coms_device_allocation.update!(allocation: 18, cap:18) # routers
+```
+
+### Missing `LocalAuthority` contact details
+
+The supplier will see the delivery details of the `LocalAuthority`. If the `LocalAuthority` does not have an address,
```
Comment (nigel-lowry) [inline docs/support/add_iss_provision.md:80]: I'm still confused over which address is what

File: docs/support/add_iss_provision.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+Review and save!
+
+```ruby
+la.save
+```
+
+### Supplier references
+
+The supplier will eventually (and manually) see the information on the new school and manually add their references
```
Comment (nigel-lowry) [inline docs/support/add_iss_provision.md:98]: Is this both `SoldTo` and `ShipTo`. How are we informed that they've done it?

File: docs/support/add_iss_provision.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+```ruby
+la.save
+```
+
+### Supplier references
+
+The supplier will eventually (and manually) see the information on the new school and manually add their references 
+within the portal.
+
+Once the supplier reference has been set we should set the school status to `can_order`.
```
Comment (nigel-lowry) [inline docs/support/add_iss_provision.md:101]: Does "supplier reference" mean `SoldTo` or `ShippedTo` or something?

File: docs/support/add_iss_provision.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+```ruby
+lafp.update!(order_state: 'can_order')
+```
+
+### Invitation to order
+
+Once the ISS has been properly configured and the supplier has added their references we can send the e-mails inviting
```
Comment (nigel-lowry) [inline docs/support/add_iss_provision.md:109]: Again, I'd like to know what references mean here. Also, do we send e-mails in code, in Notify.Gov or typed out manually by a support person?

File: docs/support/add_iss_provision.md (hunk 1 above)
Comment (markhallen) [reply docs/support/add_iss_provision.md:39]: Clarifying with this "Interested LocalAuthorities will provide delivery details for the ISS Provision. This address will be used to complete the contact details for the new LaFundedPlace object:"

File: docs/support/add_iss_provision.md (hunk 2 above)
Comment (markhallen) [reply docs/support/add_iss_provision.md:68]: We need to wait for the supplier to confirm that they have updated their system to include the new LaFundedPalce before we add the allocations.
Whether you are coming back to the same console session or starting and new console session you should ensure that you have the ``lafp`` variable set correctly:

````ruby
urn=845;lafp=School.find_by_provision_urn("ISS#{urn}")
````

Inspect the ``lafp`` variable to ensure that it correctly reflects the LaFundedPlace that you want to update.

File: docs/support/add_iss_provision.md (hunk 3 above)
Comment (markhallen) [reply docs/support/add_iss_provision.md:80]: Removing this section completely based on recent findings.

File: docs/support/add_iss_provision.md (hunk 4 above)
Comment (markhallen) [reply docs/support/add_iss_provision.md:98]: This is the shipTo which directly relates to computacenter_reference, but we want to keep the description supplier agnostic. Do you think we should expand any further?

File: docs/support/add_iss_provision.md (hunk 5 above)
Comment (markhallen) [reply docs/support/add_iss_provision.md:101]: Same as above :-)

File: docs/support/add_iss_provision.md (hunk 6 above)
Comment (markhallen) [reply docs/support/add_iss_provision.md:109]: Clarifying with "The emails are normally sent by the support team.".

File: docs/support/add_iss_provision.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+to order devices on behalf of students in independent special schools. These are typically pupils
+whose fees are being paid for them in order to attend an independent special school which can
+better cater to their needs than a state school.
+
+An ISS Provisioned `LaFundedPlace`, a type of `School`, will always have the same name 
+"State-funded pupils in independent special schools and alternative provision". In our system it's 
+modelled as a `School` but really it isn't: it's just a representation of a number of pupils at 
+different ISSs overseen by a single `LocalAuthority`. Thereby the `name` attribute has to be 
+generic. 
+
```
Comment (kenfodder) [inline docs/support/add_iss_provision.md:13]: Maybe add: "This is why a `LocalAuthority` only `has_one iss_provision` and/or `has_one scl_provision`.

**Review by kenfodder — APPROVED** [review]

File: docs/support/add_iss_provision.md (hunk 7 above)
Comment (kenfodder) [reply docs/support/add_iss_provision.md:13]: Otherwise LGTM.

**Pull request merged**